MacCoolApp/MacCoolApp/Info.plist

```diff
@@ -9,7 +9,7 @@
 	<key>CFBundleIconFile</key>
 	<string>MyIcon</string>
 	<key>CFBundleIdentifier</key>
-	<string>com.xamarin.mac-publish-workflow</string>
+	<string>com.xamarin.mac-publish-workflow-II</string>
```

Comment:
Out of curiosity what was wrong with the previous ID? Taken by another account?

Reply:
nothing wrong with the ID itself but there's two apps sharing the same one
so if you submit them you'll lose the result of the first one being processed (which might not be in the order they were submitted)
That 2nd id already existed in App Store connect - so it was an easy change